services/ingestor/src/main/resources/.env.example

```diff
@@ -0,0 +1,16 @@
+# Kafka Producer config
+producer_config_bootstrap.servers=localhost:9092
```

Comment:
is it standard to have "." in env vars?

Reply:
I have only seen upper snake cased ENV keys

Reply:
This is from Kafka for getting the producer config. To get a map of producer config, we check all envs for prefix `producer_config_`, strip that prefix and then supply that key-value pair to Kafka.

Reply:
"This is from Kafka for getting the producer config" 

Yes I got that, just curious on whether we should put effort on following standards. Since these are custom envs, we can parse and convert the casing.

Reply:
I'd suggest no transforms, bc we do not know how the producer config expects the key value pairs to be. So we should just supply it as-is.